digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/LogUtils.scala

```diff
@@ -8,13 +8,18 @@ object LogUtils {
 
   /** Logs the time it took function <i>f</i> to complete, when ever it took more than {@link timeLoggingThresholdInMs}.
     * In case <i>f</i> fails, log line is printed as an error, otherwise as debug level line. */
-  def time[R](logger: Logger, operationName: String)(f: => R): R = {
+  def time[R](logger: Logger, operationName: String, noFilter: Boolean = false, url :Option[String] = None)(f: => R): R = {
     val begin = System.currentTimeMillis()
     try {
       val result = f
       val duration = System.currentTimeMillis() - begin
-      if (duration >= timeLoggingThresholdInMs) {
-        logger.debug(s"$operationName completed in $duration ms")
+      val urlString = if ( url.isDefined) {s"URL: ${url.get}"}else ""
+      if (noFilter) {
+        logger.info(s"$operationName completed in $duration ms and in second ${duration / 1000}, ${urlString}")
```

Comment:
debug tais muuttua info,

Onko teillä jotakin systeemiä nopeasti saada tää päälle tai pois? 
vähän kankeaa tehdä uus build aina kun pitää vaihtaa loggaus tasoa

Reply:
https://examples.javacodegeeks.com/enterprise-java/logback/logback-change-log-level-runtime-example/ jotakin tälläistä vois viritellä

Reply:
https://dzone.com/articles/5-ways-to-change-the-log-levels-at-runtime

Reply:
Onko asiasta tiketti?